launcher/game/updater.rpy

```diff
@@ -71,9 +71,29 @@ init python:
     _("Nightly")
     _("The bleeding edge of Ren'Py development. This may have the latest features, or might not run at all.")
 
+    # Newly added functions. Signed, MPDS
+    # Update: GET OUT OF MY HEAD
```

Comment:
Normally comments like this is not great to see in a pro workspace or for anyone seeing the code in how it runs